### PR TITLE
Only allow unique marks (#1614)

### PR DESCRIPTION
**Pull request by Airblader**

### CODE REVIEW

**Comment by Airblader** [comment]
By the way, is there a specific reason we don't allow marks to be used on multiple windows? Sure, `[con_mark="foo"] focus` becomes a weird thing to do then, but that's up to the user. I could also do something like `[con_mark="foo"] move container to …` or any other command.

File: testcases/lib/i3test.pm [hunk 1]
```diff
@@ -614,7 +614,7 @@ Sends the specified command to i3.
 
 =cut
 sub cmd {
-    i3(get_socket_path())->command(@_)->recv
+    return i3(get_socket_path())->command(@_)->recv
```
Comment (stapelberg) [inline testcases/lib/i3test.pm]: This is unnecessary. The default return value is the last expression in Perl.

**Comment by stapelberg** [comment]
> By the way, is there a specific reason we don't allow marks to be used on multiple windows? Sure, [con_mark="foo"] focus becomes a weird thing to do then, but that's up to the user. I could also do something like [con_mark="foo"] move container to … or any other command.

Since marks were envisioned to be used for specifically such commands (that operate on a single window), it was a good thing to avoid weird use-cases :).

File: testcases/lib/i3test.pm (hunk 1 above)
Comment (Airblader) [inline testcases/lib/i3test.pm]: Unnecessary, but explicit. But okay, I'll remove it. I'll leave the description change of it, though. :)

**Comment by Airblader** [comment]
@stapelberg Alright. If we want to stick to the uniqueness then, I've incorporated your comment and rebased it.

**Pull request merged**